src/envConfig.ts

```diff
@@ -235,6 +235,14 @@ export const asEnvConfig = asObject({
       appId: asOptional(asString, 'edge')
     }).withRest
   ),
+  RANGO_INIT: asCorePluginInit(
+    asObject({
+      appId: asOptional(asString, 'edge'),
+      rangoApiKey: asOptional(asString, ''),
+      referrerAddress: asOptional(asString, ''),
+      referrerFee: asOptional(asString, '0.5')
```

Comment:
Please change to 0.75

Reply:
It seems you have already changed it in the other PR.